scripts.py

```diff
@@ -737,11 +737,11 @@ class BibliographicRefreshScript(RunCollectionCoverageProviderScript, Identifier
     )
 
     def __init__(self, _db=None, **metadata_replacement_args):
-        replacement_policy = ReplacementPolicy.from_metadata_source(
+        self.replacement_policy = ReplacementPolicy.from_metadata_source(
             **metadata_replacement_args
         )
-        kwargs = dict(replacement_policy=replacement_policy)
 
+        kwargs = dict(replacement_policy=self.replacement_policy)
```

Comment:
This is an unrelated change; I happened to run this script recently and its `do_run` method assumed the replacement policy had been saved on the Script object.